chevah_build

```diff
@@ -74,7 +84,11 @@ PYTHON_BIN=$INSTALL_FOLDER/bin/python
 PYTHON_BUILD_FOLDER="$PYTHON_VERSION-$OS-$ARCH"
 
 export MAKE=make
+
+# Used when building cffi on ArchLinux
 export CHEVAH_BUILD_PATH=$INSTALL_FOLDER
+# Used when building python on Windows or ArchLinux
+export PIP_ARGS
```

Comment:
I did not want to mess with the windows installation and PIP_ARGS is used in python/chevahbs.